packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperPlotNav.stories.tsx

```diff
@@ -36,11 +37,35 @@ const meta = {
 
 export default meta
 type Story = StoryObj<typeof meta>
-
+const handlePlotClick = (event: QueryWrapperSynapsePlotRowClickEvent) => {
+  // window.alert(`Handling click on this row: ${JSON.stringify(event.row)}`)
+  const { getCurrentQueryRequest, executeQueryRequest } = event.queryContext
+  const requestCopy = getCurrentQueryRequest()
+  requestCopy.query.additionalFilters = [
+    ...(requestCopy.query.additionalFilters || []),
+    {
+      concreteType:
+        'org.sagebionetworks.repo.model.table.ColumnSingleValueQueryFilter',
+      columnName: 'resourceType',
+      operator: ColumnSingleValueFilterOperator.EQUAL,
+      values: [event.row.values[0] as string],
+    },
+  ]
+  executeQueryRequest(requestCopy)
+}
 export const Cards: Story = {
   args: {
     name: 'Tools',
     sql: 'SELECT * FROM syn26438037',
+    customPlots: [
```

Comment:
This is a silly demo, since these particular statistics are returned in the resourceType Values facet, but it demonstrates and exercises the new functionality.